ts/packages/library/src/domain/bot.ts

```diff
@@ -119,13 +119,46 @@ export type ApiKeyJwtAuthToken = { kind: "api_jwt"; token: string };
 
 export type ApiKey = { kind: "api_key"; token: string };
 
-export type BlobReference = {
-    blobId: bigint;
-    canisterId: string;
+export type SlashCommandParamInstance = CommandParamInstance & SlashCommandParamTypeInstance;
```

Comment:
These are renamed in the Rust code:

ParamInstance -> Arg and SlashCommand -> Command

So 

SlashCommandParamInstance -> CommandArg

I think it would be good to be consistent and use the shorter names

Reply:
Done - I'll do the OC side too in a minute.